lasagne/layers/dnn.py

```diff
@@ -314,3 +315,191 @@ def get_output_for(self, input, **kwargs):
         else:
             activation = conved + self.b.dimshuffle('x', 0, 'x', 'x')
         return self.nonlinearity(activation)
+
+
+class Conv3DDNNLayer(DNNLayer):
+    """
+    lasagne.layers.Conv3DDNNLayer(incoming, num_filters, filter_size,
+    stride=(1, 1, 1), pad=0, untie_biases=False,
+    W=lasagne.init.GlorotUniform(), b=lasagne.init.Constant(0.),
+    nonlinearity=lasagne.nonlinearities.rectify, flip_filters=False,
+    **kwargs)
+
+    3D convolutional layer
+
+    Performs a 3D convolution on its input and optionally adds a bias and
+    applies an elementwise nonlinearity.  This is an alternative implementation
```

Comment:
I guess "This is an alternative implementation" doesn't make much sense considering that it's the only one right now :)